DatadogInternal/Sources/Utils/MetaTypeExtensions.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+internal enum MetaTypeExtensions {
+    static func key(from anyClass: Any) -> String {
```

Comment:
/question Why do we need to sanitize the class name?

Reply:
Yes, but it's only used as key, we don't need it to be human readable, no?
and why not using `String(describing: delegateClass)`?

Reply:
because this gibberish changes from time to time specially the `1130f7094` part. We need a consistent fully qualified name of the type.